doc/api/process.md

```diff
@@ -2158,6 +2158,14 @@ a [Writable][] stream.
 `process.stderr` differs from other Node.js streams in important ways. See
 [note on process I/O][] for more information.
 
+### `process.stderr.fd`
+
+* {number}
+
+This property refers to the value of underlying file descriptor
+(in POSIX) or handle (an abstract resource reference in Windows) of
```

Comment:
It does not refer to the handle on Windows.

The values are fixed at 0, 1 and 2, respectively.

Reply:
@addaleax, fixed it. Thanks.